clang/lib/DPCT/APINames_nvSHMEM.inc

```diff
@@ -0,0 +1,2423 @@
+ENTRY(nvshmemx_ushort_put_nbi_warp, nvshmemx_ushort_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_uint_put_nbi_warp, nvshmemx_uint_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_ulong_put_nbi_warp, nvshmemx_ulong_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_ulonglong_put_nbi_warp, nvshmemx_ulonglong_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_int8_put_nbi_warp, nvshmemx_int8_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_int16_put_nbi_warp, nvshmemx_int16_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_int32_put_nbi_warp, nvshmemx_int32_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_int64_put_nbi_warp, nvshmemx_int64_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_uint8_put_nbi_warp, nvshmemx_uint8_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_uint16_put_nbi_warp, nvshmemx_uint16_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_uint32_put_nbi_warp, nvshmemx_uint32_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_uint64_put_nbi_warp, nvshmemx_uint64_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_size_put_nbi_warp, nvshmemx_size_put_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_ptrdiff_put_nbi_warp, nvshmemx_ptrdiff_put_nbi_warp, false, NO_FLAG, P4, "comment")
+
+ENTRY(nvshmem_put8_nbi, nvshmem_put8_nbi, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmem_put16_nbi, nvshmem_put16_nbi, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmem_put32_nbi, nvshmem_put32_nbi, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmem_put64_nbi, nvshmem_put64_nbi, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmem_put128_nbi, nvshmem_put128_nbi, false, NO_FLAG, P4, "comment")
+
+ENTRY(nvshmemx_put8_nbi_on_stream, nvshmemx_put8_nbi_on_stream, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_put16_nbi_on_stream, nvshmemx_put16_nbi_on_stream, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_put32_signal_nbi_on_stream, nvshmemx_put32_signal_nbi_on_stream, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_put64_nbi_on_stream, nvshmemx_put64_nbi_on_stream, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_put128_nbi_on_stream, nvshmemx_put128_nbi_on_stream, false, NO_FLAG, P4, "comment")
+
+ENTRY(nvshmemx_put8_nbi_block, nvshmemx_put8_nbi_block, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_put16_nbi_block, nvshmemx_put16_nbi_block, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_put32_signal_nbi_block, nvshmemx_put32_signal_nbi_block, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_put64_nbi_block, nvshmemx_put64_nbi_block, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_put128_nbi_block, nvshmemx_put128_nbi_block, false, NO_FLAG, P4, "comment")
+
+ENTRY(nvshmemx_put8_nbi_warp, nvshmemx_put8_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_put16_nbi_warp, nvshmemx_put16_nbi_warp, false, NO_FLAG, P4, "comment")
+ENTRY(nvshmemx_put32_signal_nbi_warp, nvshmemx_put32_signal_nbi_warp, false, NO_FLAG, P4, "comment")
```

Comment:
Why only put32 add a signal?

Reply:
It was a misplaced replacement
Corrected it now